<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="5" total="10" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Main Suite" duration-ms="191021" started-at="2018-09-08T11:15:52Z" finished-at="2018-09-08T11:19:03Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="173015" started-at="2018-09-08T11:15:52Z" finished-at="2018-09-08T11:18:45Z">
      <class name="com.test.testcase.ForgotPasswordTestcase">
        <test-method status="PASS" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="beforeClass" is-config="true" duration-ms="3" started-at="2018-09-08T16:46:30Z" finished-at="2018-09-08T16:46:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@1a968a59]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-09-08T16:46:30Z" finished-at="2018-09-08T16:46:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.testcase.ForgotPasswordTestcase.mailidblank() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="mailidblank()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="mailidblank" duration-ms="843" started-at="2018-09-08T16:46:30Z" finished-at="2018-09-08T16:46:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mailidblank -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="afterMethod" is-config="true" duration-ms="551" started-at="2018-09-08T16:46:31Z" finished-at="2018-09-08T16:46:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=mailidblank status=SUCCESS method=ForgotPasswordTestcase.mailidblank()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@1a968a59]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2018-09-08T16:46:31Z" finished-at="2018-09-08T16:46:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.testcase.ForgotPasswordTestcase.invalidmailadress() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="invalidmailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="invalidmailadress" duration-ms="29321" started-at="2018-09-08T16:46:31Z" finished-at="2018-09-08T16:47:01Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to locate element: //*[@tabindex='1']
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'
System info: host: 'MEET', ip: '192.168.0.108', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 56.0, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 4104, moz:profile: C:\Users\Hiren Gajera\AppDa..., pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 6.3, rotatable: false, specificationLevel: 0, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: ebe6883c-259e-423e-b25f-712b6ee27e89
*** Element info: {Using=xpath, value=//*[@tabindex='1']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: //*[@tabindex='1']
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'
System info: host: 'MEET', ip: '192.168.0.108', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 56.0, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 4104, moz:profile: C:\Users\Hiren Gajera\AppDa..., pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 6.3, rotatable: false, specificationLevel: 0, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: ebe6883c-259e-423e-b25f-712b6ee27e89
*** Element info: {Using=xpath, value=//*[@tabindex='1']}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:371)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy11.clear(Unknown Source)
	at com.test.pages.ForgotPasswordPage.clearemailadress(ForgotPasswordPage.java:31)
	at com.test.testcase.ForgotPasswordTestcase.invalidmailadress(ForgotPasswordTestcase.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidmailadress -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="afterMethod" is-config="true" duration-ms="1050" started-at="2018-09-08T16:47:01Z" finished-at="2018-09-08T16:47:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=invalidmailadress status=FAILURE method=ForgotPasswordTestcase.invalidmailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@1a968a59]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2018-09-08T16:47:02Z" finished-at="2018-09-08T16:47:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.testcase.ForgotPasswordTestcase.doesnotexistmailadress() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="doesnotexistmailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="doesnotexistmailadress" duration-ms="61445" started-at="2018-09-08T16:47:02Z" finished-at="2018-09-08T16:48:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doesnotexistmailadress -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="afterMethod" is-config="true" duration-ms="884" started-at="2018-09-08T16:48:03Z" finished-at="2018-09-08T16:48:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=doesnotexistmailadress status=SUCCESS method=ForgotPasswordTestcase.doesnotexistmailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@1a968a59]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="FAIL" signature="validemailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="validemailadress" duration-ms="37953" started-at="2018-09-08T16:48:04Z" finished-at="2018-09-08T16:48:42Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Email sent to user.] but found [Invalid Email Address.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Email sent to user.] but found [Invalid Email Address.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at com.test.testcase.ForgotPasswordTestcase.validemailadress(ForgotPasswordTestcase.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validemailadress -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-09-08T16:48:04Z" finished-at="2018-09-08T16:48:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.testcase.ForgotPasswordTestcase.validemailadress() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19]" name="afterMethod" is-config="true" duration-ms="1315" started-at="2018-09-08T16:48:42Z" finished-at="2018-09-08T16:48:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validemailadress status=FAILURE method=ForgotPasswordTestcase.validemailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@1b604f19] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@1a968a59]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- com.test.testcase.ForgotPasswordTestcase -->
      <class name="com.test.testcase.HomeTestcase">
        <test-method status="PASS" signature="setUp(java.lang.String, java.lang.String)[pri:0, instance:com.test.testcase.HomeTestcase@1593948d]" name="setUp" is-config="true" duration-ms="34828" started-at="2018-09-08T16:45:52Z" finished-at="2018-09-08T16:46:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Chrome Browser]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:com.test.testcase.HomeTestcase@1593948d]" name="beforeClass" is-config="true" duration-ms="32" started-at="2018-09-08T16:46:26Z" finished-at="2018-09-08T16:46:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@1a968a59]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.testcase.HomeTestcase@1593948d]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2018-09-08T16:46:26Z" finished-at="2018-09-08T16:46:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.testcase.HomeTestcase.clickonforgotpassword() throws java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="clickonforgotpassword()[pri:0, instance:com.test.testcase.HomeTestcase@1593948d]" name="clickonforgotpassword" duration-ms="270" started-at="2018-09-08T16:46:26Z" finished-at="2018-09-08T16:46:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickonforgotpassword -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:com.test.testcase.HomeTestcase@1593948d]" name="afterMethod" is-config="true" duration-ms="3252" started-at="2018-09-08T16:46:27Z" finished-at="2018-09-08T16:46:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=clickonforgotpassword status=SUCCESS method=HomeTestcase.clickonforgotpassword()[pri:0, instance:com.test.testcase.HomeTestcase@1593948d] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@1a968a59]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestContext)[pri:0, instance:com.test.testcase.HomeTestcase@1593948d]" name="tearDown" is-config="true" duration-ms="1219" started-at="2018-09-08T16:48:43Z" finished-at="2018-09-08T16:48:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@1a968a59]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.testcase.HomeTestcase -->
    </test> <!-- Test -->
    <test name="FirefoxTest" duration-ms="191021" started-at="2018-09-08T11:15:52Z" finished-at="2018-09-08T11:19:03Z">
      <class name="com.test.testcase.ForgotPasswordTestcase">
        <test-method status="PASS" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="beforeClass" is-config="true" duration-ms="3" started-at="2018-09-08T16:46:31Z" finished-at="2018-09-08T16:46:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4667ae56]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-09-08T16:46:31Z" finished-at="2018-09-08T16:46:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.testcase.ForgotPasswordTestcase.mailidblank() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="mailidblank()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="mailidblank" duration-ms="9921" started-at="2018-09-08T16:46:31Z" finished-at="2018-09-08T16:46:41Z">
          <exception class="org.openqa.selenium.StaleElementReferenceException">
            <message>
              <![CDATA[The element reference of <div> stale: either the element is no longer attached to the DOM or the page has been refreshed
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'
System info: host: 'MEET', ip: '192.168.0.108', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 56.0, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 4104, moz:profile: C:\Users\Hiren Gajera\AppDa..., pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 6.3, rotatable: false, specificationLevel: 0, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: ebe6883c-259e-423e-b25f-712b6ee27e89]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.StaleElementReferenceException: The element reference of <div> stale: either the element is no longer attached to the DOM or the page has been refreshed
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'
System info: host: 'MEET', ip: '192.168.0.108', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 56.0, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 4104, moz:profile: C:\Users\Hiren Gajera\AppDa..., pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 6.3, rotatable: false, specificationLevel: 0, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: ebe6883c-259e-423e-b25f-712b6ee27e89
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:279)
	at org.openqa.selenium.remote.RemoteWebElement.getText(RemoteWebElement.java:160)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
	at com.sun.proxy.$Proxy11.getText(Unknown Source)
	at com.test.pages.ForgotPasswordPage.blankemailvalidationverify(ForgotPasswordPage.java:43)
	at com.test.testcase.ForgotPasswordTestcase.mailidblank(ForgotPasswordTestcase.java:18)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.StaleElementReferenceException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mailidblank -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="afterMethod" is-config="true" duration-ms="20592" started-at="2018-09-08T16:46:41Z" finished-at="2018-09-08T16:47:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=mailidblank status=FAILURE method=ForgotPasswordTestcase.mailidblank()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4667ae56]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2018-09-08T16:47:01Z" finished-at="2018-09-08T16:47:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.testcase.ForgotPasswordTestcase.invalidmailadress() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="invalidmailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="invalidmailadress" duration-ms="61405" started-at="2018-09-08T16:47:01Z" finished-at="2018-09-08T16:48:03Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[Unable to locate element: //*[contains(text(),'Invalid')]
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'
System info: host: 'MEET', ip: '192.168.0.108', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 56.0, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 4104, moz:profile: C:\Users\Hiren Gajera\AppDa..., pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 6.3, rotatable: false, specificationLevel: 0, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: ebe6883c-259e-423e-b25f-712b6ee27e89
*** Element info: {Using=xpath, value=//*[contains(text(),'Invalid')]}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: //*[contains(text(),'Invalid')]
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'
System info: host: 'MEET', ip: '192.168.0.108', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_60'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities {acceptInsecureCerts: true, browserName: firefox, browserVersion: 56.0, javascriptEnabled: true, moz:accessibilityChecks: false, moz:headless: false, moz:processID: 4104, moz:profile: C:\Users\Hiren Gajera\AppDa..., pageLoadStrategy: normal, platform: XP, platformName: XP, platformVersion: 6.3, rotatable: false, specificationLevel: 0, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}}
Session ID: ebe6883c-259e-423e-b25f-712b6ee27e89
*** Element info: {Using=xpath, value=//*[contains(text(),'Invalid')]}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:371)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy11.getText(Unknown Source)
	at com.test.pages.ForgotPasswordPage.invalidemailvalidationverify(ForgotPasswordPage.java:47)
	at com.test.testcase.ForgotPasswordTestcase.invalidmailadress(ForgotPasswordTestcase.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidmailadress -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="afterMethod" is-config="true" duration-ms="795" started-at="2018-09-08T16:48:03Z" finished-at="2018-09-08T16:48:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=invalidmailadress status=FAILURE method=ForgotPasswordTestcase.invalidmailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4667ae56]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2018-09-08T16:48:04Z" finished-at="2018-09-08T16:48:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.testcase.ForgotPasswordTestcase.doesnotexistmailadress() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="doesnotexistmailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="doesnotexistmailadress" duration-ms="38449" started-at="2018-09-08T16:48:04Z" finished-at="2018-09-08T16:48:42Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Email doesn't exists.] but found [Invalid Email Address.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Email doesn't exists.] but found [Invalid Email Address.]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at com.test.testcase.ForgotPasswordTestcase.doesnotexistmailadress(ForgotPasswordTestcase.java:46)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doesnotexistmailadress -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="afterMethod" is-config="true" duration-ms="816" started-at="2018-09-08T16:48:42Z" finished-at="2018-09-08T16:48:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=doesnotexistmailadress status=FAILURE method=ForgotPasswordTestcase.doesnotexistmailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4667ae56]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="validemailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="validemailadress" duration-ms="18393" started-at="2018-09-08T16:48:43Z" finished-at="2018-09-08T16:49:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validemailadress -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="beforeMethod" is-config="true" duration-ms="0" started-at="2018-09-08T16:48:43Z" finished-at="2018-09-08T16:48:43Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.testcase.ForgotPasswordTestcase.validemailadress() throws java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79]" name="afterMethod" is-config="true" duration-ms="758" started-at="2018-09-08T16:49:01Z" finished-at="2018-09-08T16:49:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validemailadress status=SUCCESS method=ForgotPasswordTestcase.validemailadress()[pri:0, instance:com.test.testcase.ForgotPasswordTestcase@51efea79] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4667ae56]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
      </class> <!-- com.test.testcase.ForgotPasswordTestcase -->
      <class name="com.test.testcase.HomeTestcase">
        <test-method status="PASS" signature="setUp(java.lang.String, java.lang.String)[pri:0, instance:com.test.testcase.HomeTestcase@6d00a15d]" name="setUp" is-config="true" duration-ms="37215" started-at="2018-09-08T16:45:52Z" finished-at="2018-09-08T16:46:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Firefox]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Firefox Browser]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="beforeClass(org.testng.ITestContext)[pri:0, instance:com.test.testcase.HomeTestcase@6d00a15d]" name="beforeClass" is-config="true" duration-ms="3" started-at="2018-09-08T16:46:29Z" finished-at="2018-09-08T16:46:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4667ae56]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.test.testcase.HomeTestcase@6d00a15d]" name="beforeMethod" is-config="true" duration-ms="1" started-at="2018-09-08T16:46:29Z" finished-at="2018-09-08T16:46:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.test.testcase.HomeTestcase.clickonforgotpassword() throws java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="PASS" signature="clickonforgotpassword()[pri:0, instance:com.test.testcase.HomeTestcase@6d00a15d]" name="clickonforgotpassword" duration-ms="1220" started-at="2018-09-08T16:46:29Z" finished-at="2018-09-08T16:46:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clickonforgotpassword -->
        <test-method status="PASS" signature="afterMethod(org.testng.ITestResult, org.testng.ITestContext)[pri:0, instance:com.test.testcase.HomeTestcase@6d00a15d]" name="afterMethod" is-config="true" duration-ms="795" started-at="2018-09-08T16:46:30Z" finished-at="2018-09-08T16:46:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=clickonforgotpassword status=SUCCESS method=HomeTestcase.clickonforgotpassword()[pri:0, instance:com.test.testcase.HomeTestcase@6d00a15d] output={null}]]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@4667ae56]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestContext)[pri:0, instance:com.test.testcase.HomeTestcase@6d00a15d]" name="tearDown" is-config="true" duration-ms="595" started-at="2018-09-08T16:49:02Z" finished-at="2018-09-08T16:49:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@4667ae56]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.test.testcase.HomeTestcase -->
    </test> <!-- FirefoxTest -->
  </suite> <!-- Main Suite -->
</testng-results>
